### DISCUSSION TITLE
How to fingerprint F31

### DISCUSSION THREAD

**Discussion by Duff78** [bimmerconnected/bimmer_connected#316] [Fingerprint data contributions]
Hi, Hope you are well.
I would like to share the fingerprint for my F31, but I don’t know how. Even after reading the description.
Does anyone have a simple description for me? Running HA assist on Rasp with Hassio.
Many thanks in advance.

**Comment by rikroe** [comment]
Many thanks for wanting to contribute! 
Due to some BMW API changes, we currently plan to add more to the fingerprint process to understand the new endpoints. 

When this is done, we can also have a look at a more detailed description for hassio users.

**Comment by gerard33** [comment]
In the meanwhile you can have a look here -> https://github.com/bimmerconnected/bimmer_connected/issues/313#issuecomment-910395660. That describes which addon to use (very important to use the correct one 😉) and how to get the fingerprint data using hass.io (or Home Assistant as it is called now).
